parsing.py

```diff
@@ -153,10 +153,12 @@ def url_to_shortlink(url):
         return url
     if id_and_site[2] == "question":
         return "http://{}/questions/{}".format(id_and_site[1], id_and_site[0])
+
```

Comment:
won't pass without the two lines, I'll do a sample commit for you with them missing